packages/suite/src/views/wallet/send/Outputs/Address.tsx

```diff
@@ -260,6 +273,24 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
                     return translationString('TR_ETH_ADDRESS_CANT_VERIFY_HISTORY');
                 }
             },
+            contract: async (address: string) => {
+                if (networkType === 'ethereum' && !contractAddressWarningDismissed) {
+                    if (isOnline) {
+                        const params = {
+                            descriptor: address,
+                            coin: symbol,
+                        };
+                        const result = await TrezorConnect.getAccountInfo(params);
```

Comment:
For evm accounts we now do 2 same requests. How about merging contract and checksum into one validation? `evmChecks`, `validateEvmAddress`, `evmAddressValidator`,.. checksum being first

Reply:
camel case please